src/main/java/world/bentobox/bentobox/api/configuration/WorldSettings.java

```diff
@@ -467,4 +467,61 @@ default boolean isTeleportPlayerToIslandUponIslandCreation() {
         return true;
     }
 
+
+    /**
+     * Returns all aliases for main admin command.
+     * It is assumed that all aliases are split with whitespace between them.
+     * String cannot be empty.
+     * Default value: {@code getFriendlyName() + "admin"} (to retain backward compatibility).
+     * @return String value
+     * @since 1.13.0
+     */
+    default String getAdminCommandAlias()
+    {
+        return this.getFriendlyName().toLowerCase() + "admin";
+    }
+
+
+    /**
+     * Returns all aliases for main user command.
+     * It is assumed that all aliases are split with whitespace between them.
+     * String cannot be empty.
+     * Default value: {@code getFriendlyName()} (to retain backward compatibility).
+     * @return String value
+     * @since 1.13.0
+     */
+    default String getUserCommandAlias()
+    {
+        return this.getFriendlyName().toLowerCase();
+    }
+
+
+    /**
+     * Returns sub-command for users when they execute main user command and they have an
+     * island.
+     * If defined sub-command does not exist in accessible user command list, then it will
+     * still call "go" sub-command.
+     * Default value: {@code "go"} (to retain backward compatibility)
+     * @return name of default sub-command for main command if user does have an island.
+     * @since 1.13.0
+     */
+    default String getDefaultHasIslandSubCommand()
```

Comment:
When compared to the name of the method below, this one's weird. Any ideas?

Reply:
Yea.. naming ... 

As there is a necessity for single command separation: `/island`
- one if an island exist: onIslandExistSubCommand
- one if an island does not exist: onIslandSubCommand

I am not sure how naming could be improved...